SUPPORT.md

```diff
@@ -0,0 +1,38 @@
+## Kubernetes User Support
+
+Welcome to Kubernetes! We use GitHub for tracking bugs and feature requests. 
+This isn't the right place to get support for using Kubernetes, but the following 
+resources are available below, thanks for understanding.
+
+### Stack Exchange
+
+The Kubernetes Community is active on Stack Overflow, you can post your questions there: 
+
+* [Kubernetes on Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
+
+  * Here are some tips for [about how to ask good questions](http://stackoverflow.com/help/how-to-ask).
+  * Don't forget to check to see [what's on topic](http://stackoverflow.com/help/on-topic).
+
+### Real-time Chat
+
+* [Slack](https://kubernetes.slack.com) ([registration](http://slack.k8s.io)):
+The `#kubernetes-users` and `#kubernetes-novice` channels are usual places where 
+people offer support.
+
+* Also check out the 
+[Slack Archive](http://kubernetes.slackarchive.io/) of past conversations.
+
+### Mailing Lists/Groups
+
+* [Kubernetes-users group](https://groups.google.com/forum/#!forum/kubernetes-users)
+
+
+### Documentation 
+
+* [User Documentation](https://kubernetes.io/docs/) 
+* [Troubleshooting Guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/)
+
+
```

Comment:
Not necessarily but I don't know if there is a better place for  kubectl/kubeadm issue reporters being told to file issues in the components repos. Anyway, I'd be also good with getting this initial revision in :)